spec/support/mockers/mocker.rb

```diff
@@ -0,0 +1,70 @@
+require 'rspec/mocks/standalone'
+
+class Mocker < RSpec::Mocks::InstanceVerifyingDouble
+  include RSpec::Mocks
+
+  FIND_METHODS = [:find, :find_by_id]
+
+  def initialize(traits: [], **attrs)
+    # from RSpec::Mocks::ExampleMethods
+    # combination of #declare_verifying_double and #declare_double
+    ref = ObjectReference.for(self.class.klass_name)
+    RSpec::Mocks.configuration.verifying_double_callbacks.each do |block|
+      block.call(ref)
+    end
+    attrs ||= {}
+    super(ref, attrs)
+    self.as_null_object
```

Comment:
To make the mockers behave as spies instead of doubles! This decision was motivated by [this](https://robots.thoughtbot.com/a-closer-look-at-test-spies) blog post.